### PR TITLE
minor changes

### PR DESCRIPTION
**Pull request by SrWhiteRD**
Ran into it. Maybe you should on those semantics. Have someone review it. Keep that pace guys.

### CODE REVIEW

**Comment by ar-x** [comment]
@SrWhiteRD Feel free to ping people on the Spanish contributors list (the original authors of this) to get this reviewed.

**Comment by SrWhiteRD** [comment]
Perfect will do.

**Comment by SrWhiteRD** [comment]
@euricanti, @mrrovot, @adj-gh can any of you review this please.

File: _i18n/es.yml [hunk 1]
```diff
@@ -49,76 +49,76 @@ index_page:
...
+  story_4: La verdad es que la mayoría de nosotros somos programadores a tiempo completo en otros proyectos, y TurtleCoin es nuestra forma de devolver algo de diversión a la comunidad.
   story_5: Cómo lo comenzamos
-  story_6: Una fría noche de diciembre, RockSteady y Bebop estaban tomando cerveza mientras discutían sobre la locura en torno a los ICO falsos y sin ningún producto. Bebop conjeturó que simplemente no sería posible lanzar un proyecto real ahora sin un equipo serio, haciendo marketing serio y aportando fondos serios.
-  story_7: RockSteady se inclinó, acarició su barba atentamente, y solo tuvo una respuesta.
-  story_8: <strong> "Sostenga mi cerveza". </strong>
-  story_9: Esa noche, 9 de diciembre de 2017, TurtleCoin nació.
+  story_6: Una fría noche de diciembre, RockSteady y Bebop estaban tomando cerveza mientras discutían sobre la locura en torno a los ICO's falsos sin ningún producto. Bebop conjetó que en este tiempo no sería posible lanzar un proyecto real sin un equipo serio, haciendo marketing serio y aportando fondos seriamente.
+  story_7: RockSteady se inclinó, acarició su barba delicadamente, y solo tuvo una respuesta:
```
Comment (ar-x) [inline _i18n/es.yml]: @SrWhiteRD The build is failing on this one. YAML is sensitive to `:`. Can you quote this entire sentence. Like:

```yaml
story_7: "RockSteady se inclinó, acarició su barba delicadamente, y solo tuvo una respuesta:"
```

Comment (SrWhiteRD) [reply _i18n/es.yml]: Fixed. @ar-x  missed it.

**Comment by ar-x** [comment]
@SrWhiteRD Deployed. Thanks!

**Pull request merged**